app/labor/markdown_fixer.rb

```diff
@@ -14,7 +14,7 @@ def add_quotes_to_title(markdown)
       markdown.gsub(/-{3}.*?-{3}/m) do |front_matter|
         front_matter.gsub(/title:\s?(.*?)(\r\n|\n)/m) do |target|
           # $1 is the captured group (.*?)
-          captured_title = $1
+          captured_title = Regexp.last_match(1)
```

Comment:
Mmm, I think this should be `Regexp.last_match(0)`

`$1` is the first matching group, see [special global variables](https://ruby-doc.org/core-2.6.1/Regexp.html#class-Regexp-label-Special+global+variables):

> $1, $2 and so on contain text matching first, second, etc capture group;

`Regex.last_match(1)` instead returns the match at the index `1` which is the second match, not the first one. The correct change should be `Regex.last_match(0)`

Reply:
@rhymes @keshavbiswa Here the `$1` and the `Regexp.last_match(1)` will be the value of the captured group `(.*?)` (the title w/o "title:") which we need. The `Regexp.last_match(0)` will return the whole string captured by `/title:\s?(.*?)(\r\n|\n)/m` (e.g. `title: The Way Through the Woods\n`) which is not we want here.

Reply:
Chaning to `Regexp.last_match(0)` led to failing `markdown_fixer` and related ones.

Reply:
I see. Will revert the changes right away.